types/migrations/20220205042039_init.sql

```diff
@@ -37,6 +39,16 @@ create table document (
   audio_slice_id uuid references media_slice (id) on delete set null
 );
 
+create table chapter (
+  id autouuid primary key,
+  chapter_title text not null,
+  document_title text,
+  author text,
+  document_id text,
```

Comment:
Don't worry too much about the insert for now, we'll have to tweak that when we write the code for data ingestion anyway. FYI tho The variable placeholder will still work with a `uuid`, yes. Another option is to pass in document short name like `WJ20` as $1 and then instead of doing `values ($1, ...)` you can do `select document.id, $2, $3 from document where document.short_name = $1` or similar.

Reply:
If you check out the [examples here](https://www.postgresql.org/docs/current/sql-insert.html) there's some `insert into ... select ...` examples.